lib/github-auth.js

```diff
@@ -205,9 +206,11 @@ function rmGithubToken(token) {
 // Personal tokens allow access to GitHub private repositories.
 // You can manage your personal GitHub token at
 // <https://github.com/settings/tokens>.
-if (serverSecrets && serverSecrets.gh_token) {
-  addGithubToken(serverSecrets.gh_token);
-}
+tokensPromise.then(function(tokens) {
+  if (serverSecrets && serverSecrets.gh_token) {
+    addGithubToken(serverSecrets.gh_token);
+  }
+});
```

Comment:
You're right, I'll add a `.catch()`.

The `then()` is needed, because `autosave()` returns a Promise afaik (https://github.com/badges/shields/commit/7c8b0e3d32ee6f33d1568d9278f0cb053224edc8).

Now that I look more closely, this code (that adds a personal token by configuration from `secrets.json`) can also be put in the `then()` after `autosave()`. In that case, this `then()` is not needed anymore.